src/server/db/tables/home.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  type InferInsertModel,
+  type InferSelectModel,
+  relations,
+} from 'drizzle-orm';
+import {
+  boolean,
+  integer,
+  pgTable,
+  primaryKey,
+  serial,
+  varchar,
+} from 'drizzle-orm/pg-core';
+import { files } from './files';
+import { localesEnum } from './locales';
+
+const homeCarouselSlides = pgTable('home_carousel_slides', {
+  id: serial('id').primaryKey(),
+  active: boolean('active').notNull(),
+  imageId: integer('image_id').references(() => files.id, {
+    onDelete: 'no action',
```

Comment:
We cannot delete images if this is set to 'no action'
```suggestion
    onDelete: 'set null',
```